packages/spark/sodasql/dialects/spark_dialect.py

```diff
@@ -58,21 +202,35 @@ def sql_tables_metadata_query(self, limit: Optional[int] = None, filter: str = N
     def sql_tables_metadata(self, limit: str = 10, filter: str = None):
         # TODO Implement limit
         with self.create_connection().cursor() as cursor:
-            cursor.execute(f"SHOW TABLES FROM {self.database};")
+            cursor.execute(f"SHOW TABLES FROM {self.database}")
```

Comment:
Pyodbc does not like the trailing `;`. I think hive requires it though. In dbt spark they implemented a connection specific [wrapper](https://github.com/dbt-labs/dbt-spark/blob/3980e5ce4bce11e96dccf47b89f48e4ba9c2de1d/dbt/adapters/spark/connections.py#L257) to handle this. Maybe we should do the same.